ThirdParty/GNUmakefile

```diff
@@ -0,0 +1,95 @@
+#
+# Build configuration
+#
+
+COMP            = gcc
+NVCC            = nvcc
+DEBUG           = FALSE
+USE_CUDA        = FALSE
+USE_KLU         = FALSE
+
+#SS_VERSION=5.7.2
+SS_VERSION=5.4.0
+
+#include $(AMREX_HOME)/Tools/GNUMake/Make.defs
+
+tp_suffix=$(COMP)$(DebugSuffix)$(CUDASuffix)$(KLUSuffix)
```

Comment:
Are we sure this is defined here ? most of these $() are not defined yet

Reply:
Ok I think I get that one too

Reply:
GNUmake is weird in that it passed through the entire file defining variables before running anything else.